api/v1alpha1/nnfstorageprofile_types.go

```diff
@@ -35,14 +35,19 @@ type NnfStorageProfileLustreCmdLines struct {
 	// Use the --mkfsoptions argument to specify the zfs create options.  See zfsprops(7).
 	// Use the --mountfsoptions argument to specify persistent mount options for the lustre targets.
 	Mkfs string `json:"mkfs,omitempty"`
+
+	// MountTarget specifies the mount command line for the lustre target.
+	// For persistent mount options for lustre targets, do not use this array; use the --mountfsoptions
+	// argument to mkfs.lustre instead.
+	MountTarget string `json:"mountTarget,omitempty"`
```

Comment:
I'm not clear on the different purposes of the MountTarget here, and the MountTarget in the next struct.